web/src/views/SubmissionPortal/Components/IntroBlurb.vue

```diff
@@ -0,0 +1,59 @@
+<script lang="ts">
+import {
+  defineComponent,
+} from '@vue/composition-api';
+
+export default defineComponent({
+  components: {},
+  setup() {
+    return {};
+  },
+});
+</script>
+
+<template>
+  <v-container class="mt-1">
+    <v-row>
+      <v-col class="px-0">
+        <h2 class="blue--text text-h6">
+          Making it easy to follow standards
+        </h2>
+        <p class="text-justify mb-0 text-body-1">
+          The Submission Portal is a flexible, template-driven tool designed to lower the barrier to collecting and reporting cohesive, standardized metadata about studies, samples, and assays. The standards we leverage include:
+        </p>
+      </v-col>
+    </v-row>
+    <v-row>
+      <v-col class="px-0">
+        <h2 class="blue--text text-h6">
+          Supporting FAIR data
+        </h2>
+        <p class="text-justify mb-0 text-body-1">
+          The Submission Portal leverages validation functions of the
+          <a
+            href="https://github.com/cidgoh/DataHarmonizer"
+            target="_blank"
+            title="View cidgoh/DataHarmonizer on GitHub"
+          >DataHarmonizer</a>
+          tool to check entered metadata values against the standards in the
+          <a
+            href="https://github.com/microbiomedata/nmdc-schema"
+            target="_blank"
```

Comment:
add `rel = noreferrer nopener` for extra security.

Reply:
Done in commit 794d5b95eef58a33da7c4cbe32cc04e40c6c2778.